lumigator/python/mzai/backend/settings.py

```diff
@@ -88,7 +88,7 @@ def RAY_WORKER_GPUS_FRACTION(self) -> float:  # noqa: N802
     def PIP_REQS(self) -> list:  # noqa: N802
         return [
             line.strip()
-            for line in Path.open("lumigator/python/mzai/lm_buddy/requirements.txt")
+            for line in Path.open("lumigator/python/mzai/evaluator/requirements.txt")
```

Comment:
Note that the way these requirements are currently pulled in means we build a pex executable that can't find them in Coreweave, modifications to come later. cc @chainlink